recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 15b2cc15f95a476fbe0623fd8f771e1e771310bf7a01f95412a0b605b6e47510
+  sha256: d7366c2e2293314112dcf2432108428a67b927b00005619feefc310d12d833f3
 
 build:
   number: 0
```

Comment:
Could you update the scripts as:
`  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv`

Reply:
done